default.xml

```diff
@@ -177,7 +177,7 @@
   <project path="external/stlport" name="CyanogenMod/android_external_stlport" />
```

Comment:
All PRs should be submitted as Gerrit Changes. Please see the documentation at https://wiki.cyanogenmod.org/w/Doc:_using_gerrit